### PR TITLE
Publish Project

### PR DESCRIPTION
**Pull request by openbots-ff**
Projects can now be published as nuget packages.
Additionally, All gradient ThemedForms have been swapped with UIForms for a cleaner look

### CODE REVIEW

**Review by openbots-ch — APPROVED** [review]

File: OpenBots.Studio/UI/Forms/frmScriptEngine.cs [hunk 1]
```diff
@@ -435,12 +435,14 @@ private void UpdateUI(string mainLogoText)
 
                 if (mainLogoText.Contains("(error)"))
                 {
+                    //BackColor = Color.OrangeRed;
```
Comment (openbots-ab) [inline OpenBots.Studio/UI/Forms/frmScriptEngine.cs:438]: Please remove this comment.

File: OpenBots.Studio/UI/Forms/frmScriptEngine.cs [hunk 2]
```diff
@@ -435,12 +435,14 @@ private void UpdateUI(string mainLogoText)
...
                     Theme.BgGradientStartColor = Color.OrangeRed;
                     Theme.BgGradientEndColor = Color.OrangeRed;
                     Invalidate();
                 }
                 else if (mainLogoText.Contains("(success)")) 
                 {
+                    //BackColor = Color.Green;
```
Comment (openbots-ab) [inline OpenBots.Studio/UI/Forms/frmScriptEngine.cs:445]: Remove this as well.

**Review by openbots-ab — CHANGES_REQUESTED** [review]
Please remove a couple of comments.

File: OpenBots.Studio/UI/Forms/frmScriptEngine.cs (hunk 1 above)
Comment (openbots-ff) [reply OpenBots.Studio/UI/Forms/frmScriptEngine.cs:438]: done

File: OpenBots.Studio/UI/Forms/frmScriptEngine.cs (hunk 2 above)
Comment (openbots-ff) [reply OpenBots.Studio/UI/Forms/frmScriptEngine.cs]: done

**Review by openbots-ab — APPROVED** [review]

**Review by openbots-ch — APPROVED** [review]

**Pull request merged**